packages/zoe/src/contracts/callSpread/pricedCallSpread.js

```diff
@@ -23,12 +23,15 @@ import { Position } from './position';
 const PERCENT_BASE = 100n;
 const BASIS_POINTS = 10000n;
 
+const evaluateInvitationInput = x =>
+  Number(x).toString().length < 4 ? (x * BASIS_POINTS) / PERCENT_BASE : x;
```

Comment:
We shouldn't interpret the input differently depending on length.  `2500` means 2500 basis points, and `50` should mean 50 basis points.

Reply:
👍